src/main/java/com/emt/controller/MatchController.java

```diff
@@ -0,0 +1,46 @@
+package com.emt.controller;
+
+import com.emt.model.exception.PlayerNotFoundException;
+import com.emt.model.request.CreateMatchRequest;
+import com.emt.model.response.CreateMatchResponse;
+import com.emt.service.MatchService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/matches")
+@RequiredArgsConstructor
+public class MatchController {
+
+  private final MatchService matchService;
+
+  @PostMapping("/report")
+  public String reportMatch(
+      @Valid @ModelAttribute CreateMatchRequest matchRequest,
+      @RequestParam Long winnerId,
+      @RequestParam Long loserId,
```

Comment:
You don't need these request parameters, there's a `winnerId` present in `CreateMatchRequest`